cast-receiver/scss/_typography.scss

```diff
@@ -0,0 +1,76 @@
+@import "colors";
+
+body {
+  color: $foreground-secondary;
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1rem;
+
+  .title {
+    color: $foreground-primary;
+    font-size: 3rem;
+
+    span.subtitle {
+      margin-left: 0.5rem;
+      font-size: 2rem;
+      font-weight: 400;
+    }
+  }
+
+  .caption {
+    font-size: 1.5rem;
+  }
+}
+
+/* latin-ext */
+@font-face {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local("Source Sans Pro Regular"), local("SourceSansPro-Regular"),
+    url(https://fonts.gstatic.com/s/sourcesanspro/v13/6xK3dSBYKcSV-LCoeQqfX1RYOo3qNq7lqDY.woff2)
+      format("woff2");
+  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB,
+    U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=5670181)